// This file describes the protobuf for attacking services

syntax = "proto3";

package attack_service_api_proto;

import "message_definitions.proto";

// Provides API for attacks on digital signatures
service DigitalSignatureAttackService {
  /**
   * Provides function for attack on ECDSA nonce reuse vulnerability
   */
  rpc ecdsaReusedNonceAttack(ReusedNonceAttackRequest)
                                      returns (ReusedNonceAttackResponse) {};
}
